streamclient/kafka/client.go

```diff
@@ -53,7 +53,7 @@ func NewClient(options ...func(*Client)) stream.Client {
 	c.ProducerConfig.Producer.RequiredAcks = sarama.WaitForAll
 	c.ProducerConfig.Producer.Retry.Max = 10
 	c.ProducerConfig.Producer.Return.Errors = true
-	c.ProducerConfig.Version = sarama.V0_10_2_0
+	c.ProducerConfig.Version = sarama.V1_14_0
```

Comment:
I think this refers to the Kafka version, and 1.14.0 is not a valid version.

See: https://github.com/Shopify/sarama/blob/master/utils.go

Reply:
I think he is right.

Reply:
You are right, thanks! I've updated the PR to default to `V0_10_2_0`. Note you can still override this setting in the config func.